ui/remoteviewwidget.cpp

```diff
@@ -899,14 +899,14 @@ QTouchEvent::TouchPoint RemoteViewWidget::mapToSource(const QTouchEvent::TouchPo
 
 #if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
 
-#if QT_VERSION > QT_VERSION_CHECK(6, 3, 0)
-    QMutableEventPoint::update(point, p);
-#define SET_POINT_VALUE(func, val) \
-    QMutableEventPoint::func(p, (val))
-#else
+#if QT_VERSION < QT_VERSION_CHECK(6, 3, 0)
     QMutableEventPoint mut = QMutableEventPoint::constFrom(point);
 #define SET_POINT_VALUE(func, value) \
     mut.func(value)
+#else
+    QMutableEventPoint::update(point, p);
+#define SET_POINT_VALUE(func, val) \
+    QMutableEventPoint::func(p, (val))
```

Comment:
Even though it made for a bigger change, I decided to reverse the two branches here, to match the rest of the QMutableEventPoint code (in the other file).

Every other use of `QMutableEventPoint` as a type is protected by an `#if QT_VERSION < QT_VERSION_CHECK(6, 3, 0)`, so having this **one** bit of code use `#if QT_VERSION >= QT_VERSION_CHECK(6, 3, 0)` instead, and reverse the order of the subsequent branches, was just confusing.